src/app/pages/component-sidenav/component-sidenav-can-load-guard.ts

```diff
@@ -0,0 +1,16 @@
+import {Injectable} from '@angular/core';
+import {CanActivate, ActivatedRouteSnapshot, Router, RouterStateSnapshot} from '@angular/router';
+import {SECTIONS} from '../../shared/documentation-items/documentation-items';
+
+@Injectable()
+export class CanActivateComponentSidenav implements CanActivate {
+  constructor(private router: Router) {}
+
+  canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot) {
+    let sectionFound = Object.keys(SECTIONS).find(
```

Comment:
Add comment explaining what this is doing? E.g. `Prevent navigation to a section if...`